about.html

```diff
@@ -0,0 +1,194 @@
+  rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
+  crossorigin="anonymous">
+    <!-- icons -->
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css">
+
+
+</head>
+
+
+<body class="body">
+  <header class="header">
+    <nav>
+      <div class="navbar navbar-expand-lg navbar-dark bg-dark navbar-social">
+        <div class="container-fluid ">
+          <div class="collapse navbar-collapse justify-content-end navbar-nav-social" id="navbarNav">
+            <ul class="navbar-nav nav-bar-items-social">
+              <li class="nav-item">
+                <a class="nav-link active nav-icon" aria-current="page" href="#"><i class="bi bi-facebook"></i></a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active" href="#"><i class="bi bi-twitter"></i></a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">Spanish</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">MyPage</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link active nav-social-text" href="#">Logout</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      
+      <div class="navbar navbar-expand-lg bg-body-tertiary navbar-menu">
+        <div class="container-fluid">
+          <div class="d-flex nav-bar-about">
+            <a class="navbar-brand" href="./index.html"><img src="https://wttwebcmsprod.blob.core.windows.net/eventimages/Singapore_Smash_2022.png" class="wttlogo"></a>
+            <ul class="navbar-nav nav-menu">
+              <li class="nav-item nav-item-home">
+                <a class="nav-link link-menu about-link" aria-current="page" href="./index.html">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#program">Program</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#join">Join</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="./index.html#speakers">Players</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu" href="#">News</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link link-menu link-campaign" href="#">LIVE</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+
+  <main class="main">
+    <section class="intro-about">
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done, but the `"About"` page elements are `stuck` to the `left`, fix them and give each element the correct style as shown in [Behance's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼  The pages should look almost identical to the original design, outside the content differences and also use the exact same `colors`, for `text`, `backgrounds`, `borders`, etc, from the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

![image](https://github.com/gdvl777/CapstoneSmash/assets/63915024/c431bc8f-5fbe-4ef6-b442-ddca243ac1f1)